### PR TITLE
CB-553 - gcp python recipe test

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java [hunk 1]
```diff
@@ -2,69 +2,62 @@
...
+    private SSHClient createSshClient(String host, String sshUser) {
+        try {
+            SSHClient client = new SSHClient();
+            client.addHostKeyVerifier(new PromiscuousVerifier());
+            client.connect(host, 22);
+            client.setConnectTimeout(commonCloudProperties.getSshTimeout());
+            client.authPublickey(sshUser, commonCloudProperties.getSshPrivateKeyPath());
+            return client;
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
```
Comment (biharitomi) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java]: I would log here something about what is going on...

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java [hunk 2]
```diff
@@ -2,69 +2,62 @@
...
         LOGGER.info("Waiting to SSH command to be executed...");
         try (Session session = startSshSession(ssh);
                     Command cmd = session.exec(command);
                     OutputStream os = IOUtils.readFully(cmd.getInputStream())) {
             LOGGER.info("The following SSH command is going to be executed on host {}: {}", ssh.getConnection().getTransport().getRemoteHost(), command);
             cmd.join(10L, TimeUnit.SECONDS);
-            return Pair.of(cmd.getExitStatus(), os.toString());
+            return os.toString();
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
```
Comment (biharitomi) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java]: The same for here...

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java (hunk 1 above)
Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java]: Not needed to be logged. Exception is logged by default.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java (hunk 2 above)
Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java]: Not needed to be logged. Exception is logged by default.

**Pull request merged**